### PR TITLE
Adding Intel Card, Dell PowerEdge R220 and Extreme Networks x460 48p/t

### PR DESCRIPTION
**Pull request by iriseden**
Adding few elements :

- Dell PowerEdge R220
- Intel X520-DA2
- Extreme Networks Summit x460 48t
- Extreme Networks Summit x460 48p

### CODE REVIEW

File: device-types/Extreme Networks/summit-x460-48p.yml [hunk 1]
```diff
@@ -0,0 +1,333 @@
+---
+manufacturer: Extreme Networks
+model: Summit X460-48p
+slug: extreme-networks-summit-x460-48p
+part_number: '16404'
+u_height: 1.0
+is_full_depth: true
+subdevice_role: parent
```
Comment (danner26) [inline device-types/Extreme Networks/summit-x460-48p.yml:8]: Why is this here? This definition is for `device-bays` where a device has a child that slots into it, or itself is a child that slots into another device. If this is not the case, please remove this definition

File: device-types/Extreme Networks/summit-x460-48t.yml [hunk 2]
```diff
@@ -0,0 +1,237 @@
+---
+manufacturer: Extreme Networks
+model: Summit X460-48t
+slug: extreme-networks-summit-x460-48t
+part_number: '16402'
+u_height: 1.0
+is_full_depth: true
+subdevice_role: parent
```
Comment (danner26) [inline device-types/Extreme Networks/summit-x460-48t.yml:8]: Why is this here? This definition is for `device-bays` where a device has a child that slots into it, or itself is a child that slots into another device. If this is not the case, please remove this definition

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Extreme Networks/summit-x460-48p.yml (hunk 1 above)
Comment (iriseden) [reply device-types/Extreme Networks/summit-x460-48p.yml]: I've removed **subdevice_role: parent** and **is_full_depth: true**, for the rest, it's useful (I think).

File: device-types/Extreme Networks/summit-x460-48t.yml (hunk 2 above)
Comment (iriseden) [reply device-types/Extreme Networks/summit-x460-48t.yml]: I've removed **subdevice_role: parent** and **is_full_depth: true**, for the rest, it's useful (I think).

**Review by harryajc — DISMISSED** [review]

**Review by harryajc — APPROVED** [review]

**Review by harryajc — APPROVED** [review]
